apps/client/scripts/sttr_build/index.js

```diff
@@ -62,10 +57,10 @@ function checkStatus(res) {
 }
 
 (async () => {
-  const permitIds = await fetch(listUrl, { headers })
+  fetch(listUrl, { headers })
```

Comment:
Do you need `async` when you remove `await`?

Reply:
Yes there is an await few lines below.